airbyte-ci/connectors/pipelines/pipelines/cli/airbyte_ci.py

```diff
@@ -151,6 +153,28 @@ def check_local_docker_configuration():
             f"Your docker daemon is configured with less CPUs than your local machine ({docker_cpus_count} vs. {local_cpus_count}). This may slow down the airbyte-ci execution. Please consider increasing the number of CPUs allocated to your docker daemon in the Resource Allocation settings of Docker."
         )
 
+def is_dagger_run_enabled_by_default() -> bool:
+    dagger_run_by_default = [
+        ["connectors", "test"],
+        ["connectors", "build"],
+        ["test"],
+        ["metadata_service"],
+    ]
```

Comment:
Do we need to do this? Seems like another thing to maintain. Can everything have dagger run enabled by default?

Reply:
Huh, I didn't realize this - how does this happen in the current behavior?

Reply:
Its through the TUI opt out list

https://github.com/airbytehq/airbyte/blob/95afbb27f61d5a2c7fac537ba850499e234cf553/airbyte-ci/connectors/pipelines/pipelines/cli/dagger_run.py#L27

Reply:
Aha! TIL. Ok, let's stick with the current behavior for now 👍🏻 we can reconsider in the future if we want to